.github/workflows/remove-lockdown-label.yml

```diff
@@ -0,0 +1,74 @@
+name: Remove Lockdown Label from PRs
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
Is there a merged option to make sure it isn't closed without merging?

Reply:
Ah, I guess that's supposed to be covered by the job below

Reply:
Yes, there is no per se "merged" option but that is the closes way